bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/LedgerMetadataSerDe.java

```diff
@@ -142,15 +163,9 @@ public LedgerMetadataFormat buildProtoFormat(LedgerMetadata metadata) {
             break;
         }
 
-        /** Hack to get around fact that ctime was never versioned correctly */
-        if (LedgerMetadataUtils.shouldStoreCtime(metadata)) {
-            builder.setCtime(metadata.getCtime());
-        }
-
-        if (metadata.hasPassword()) {
-            builder.setDigestType(apiToProtoDigestType(metadata.getDigestType()))
-                .setPassword(ByteString.copyFrom(metadata.getPassword()));
-        }
+        builder.setCtime(metadata.getCtime());
+        builder.setDigestType(apiToProtoDigestType(metadata.getDigestType()))
+            .setPassword(ByteString.copyFrom(metadata.getPassword()));
```

Comment:
If password is empty, we could use `ByteString.EMPTY`

Reply:
done